### ISSUE TITLE
Submenu toggles are missing initial `aria-expanded` attributes

### ISSUE THREAD

**Issue by schlessera** [WordPress/twentytwenty#850]
The submenu toggles in the mobile menu are missing the initial `aria-expanded` attribute.

Once the toggle is activated, the `aria-expanded` attribute is added to be `true`.

![Image 2019-10-14 at 2 41 32 PM](https://user-images.githubusercontent.com/83631/66754242-d73abf00-ee95-11e9-8f69-04643716a177.png)

**Comment by carolinan** [comment]
I should be able to add this to #834

**Comment by ianbelanger79** [comment]
I'm not sure that #852 fixes this. The sub-menu still does not have the `aria-expanded` attribute before the sub-menu is opened. @schlessera could you confirm this?

**Comment by schlessera** [comment]
@ianbelanger79 I understood @carolinan's latest comment as an intention, not something that has been done yet. So, I think the required change just hasn't been included yet.

**Comment by carolinan** [comment]
Hm? The sub **menu** should not have it. The toggle **buttons** should: 
https://github.com/WordPress/twentytwenty/pull/852/files#diff-1ed8515edad4563d48149dac5b5070c0R133


(The horizontal menu does not have a true toggle button.)

**Comment by schlessera** [comment]
Ah, only noticed now that your original comment was updated...
